packages/blade/src/components/BottomSheet/BottomSheet.web.tsx

```diff
@@ -433,31 +412,43 @@ const _BottomSheet = ({
 
   return (
     <BottomSheetContext.Provider value={contextValue}>
-      <BottomSheetBackdrop zIndex={zIndex} />
-      <BottomSheetSurface
-        {...metaAttribute({ name: MetaConstants.BottomSheet })}
-        {...makeAccessible({ modal: true, role: 'dialog' })}
-        windowHeight={dimensions.height}
-        isDragging={isDragging}
-        style={{
-          opacity: isVisible ? 1 : 0,
-          pointerEvents: isVisible ? 'all' : 'none',
-          height: positionY,
-          bottom: 0,
-          top: 'auto',
-          zIndex,
-        }}
-      >
-        <BaseBox height="100%" display="flex" flexDirection="column">
-          <BottomSheetGrabHandle
-            ref={grabHandleRef}
-            isHeaderFloating={isHeaderFloating}
-            {...metaAttribute({ name: ComponentIds.BottomSheetGrabHandle })}
-            {...bind()}
-          />
-          {children}
-        </BaseBox>
-      </BottomSheetSurface>
+      {isMounted ? (
```

Comment:
Why do we need to conditionally render this?

Reply:
Oh okay. @anuraghazra can you cross check this once?

Reply:
This might break integration with Dropdown, check the above check where we have `!isMounted && !isInsideDropdown`

The bottomsheet needs to be rendered and not unmount when using with Dropdown.

Reply:
@anuraghazra Oh, ok. Do you think we can use ```isOpen``` ?

Reply:
@anuraghazra So you mean when ```!isMounted == true``` and ```!isInsideDropdown == false``` in that case we should not return ```null``` in the conditional rendering?

Reply:
@archie252000 never mind that logic, this is for an edge case where imagine the bottomsheet has a input inside of it and you close the bottomsheet while being focused on the input in that case we want to destroy the react tree so the input is blurred. 

Your code should work as is.

Reply:
@anuraghazra I found a solution that is passing the test for ```<BottomSheet /> › should compose with Dropdown multi select```. Basically Conditionally wrapping the contents of BottomSheet with FloatingFocusManager instead of conditionally rendering the whole component. What do you think about that?